metriq_gym/benchmarks/benchmark.py

```diff
@@ -6,11 +6,21 @@
 from qbraid import GateModelResultData, QuantumDevice
 
 
+"""
```

Comment:
Do we want to put these comments as docstrings for the classes themselves as opposed to block comments above the classes?